anvio/splitter_matt.py

```diff
@@ -0,0 +1,1378 @@
+# -*- coding: utf-8
```

Comment:
I feel like this file shouldn't be here. I thought @ekiefl had removed it. is this keep coming back?

Reply:
Removed the file, I must of not used `git add` when I originally tried to remove the file